README.md

```diff
@@ -93,39 +92,38 @@ import { Novu } from '@novu/node';
 
 const novu = new Novu(process.env.NOVU_API_KEY);
 
-await novu.trigger('<TRIGGER_NAME>',
-  {
-    to: [{
+await novu.trigger('<TRIGGER_NAME>', {
+  to: [
+    {
       subscriberId: '<UNIQUE_IDENTIFIER>',
       email: 'john1@doemail.com',
       firstName: 'John',
       lastName: 'Doe',
-    }],
-    payload: {
-      name: "Hello World",
-      organization: {
-        logo: 'https://happycorp.com/logo.png',
-      },
     },
-  }
-);
+  ],
+  payload: {
+    name: 'Hello World',
+    organization: {
+      logo: 'https://happycorp.com/logo.png',
+    },
+  },
+});
 ```
 
 ## Embeddable Notification Center
 
-Using the Novu API and admin panel you can easily add real-time notification center to your web-app without the hassle of building it yourself. You can use our React component or an iframe embed if you are not using React.
+Using the Novu API and admin panel you can easily add real-time notification center to your web-app without the hassle of building it yourself. You can use our React component or an iFrame embed if you are not using React.
```

Comment:
```suggestion
Using the Novu API and admin panel you can easily add real-time notification center to your web-app without the hassle of building it yourself. You can use our React component or an iframe embed if you are not using React.
```
Arguably this is an opinionated capitalisation that is not used by the W3C.
https://www.w3.org/TR/2011/WD-html5-20110525/the-iframe-element.html